contao/core/2014-04-07.yaml

```diff
@@ -2,16 +2,10 @@ title:     Insufficient input validation allows for code injection and remote ex
 link:      https://contao.org/en/news/new-security-hole-found-in-contao.html
 cve:       ~
 branches:
-    2.11.x:
```

Comment:
Same: this CVE is an older one - why are you updating it instead of just adding a new file?